util/travis_utils.sh

```diff
@@ -11,7 +11,7 @@ TRAVIS_COMMIT_RANGE="${TRAVIS_COMMIT_RANGE:-HEAD~1..HEAD}"
 
 # Extra variables
 LOCAL_BRANCH=$(git rev-parse --abbrev-ref HEAD)
-QMK_CHANGES=$(git diff --name-only -n 1 ${TRAVIS_COMMIT_RANGE})
+QMK_CHANGES=$(git diff --name-only --diff-filter=d -n 1 ${TRAVIS_COMMIT_RANGE})
```

Comment:
Saw that CI was failing on trying to build the renamed keymap, so removed deleted files from `QMK_CHANGES`.